frontend/components/Section/Section.tsx

```diff
@@ -0,0 +1,120 @@
+import { useState, useEffect } from "react";
+import ImageSlider from "@/components/InteractiveImage/ImageSlider";
+import Image from "next/image";
+import RawHtml from "@/components/RawHtml/RawHtml";
+
+export type Content = {
+  id?: string;
+  type?: string;
+  value?: Slider | StaticImage | string;
+};
+
+export type Slider = {
+  id?: number;
+  name?: string;
+  currentValue?: number;
+  sliderValueDefault?: number;
+  sliderValueMax?: number;
+  sliderValueMin?: number;
+};
+
+export type StaticImage = {
+  id?: number;
+  title?: string;
+  img?: {
+    alt: string;
+    height: number;
+    width: number;
+    src: string;
+  };
+};
+
+export default function Section({ data }) {
+  const [value, setValue] = useState<number>(0);
+  const [content, setContent] = useState<Content[]>([]);
+  const [media, setMedia] = useState<StaticImage>({});
+
+  useEffect(() => {
+    const contentArr: Content[] = [];
+    data?.value.content.map((content: Content) => {
+      switch (content.type) {
+        case "slider":
+        case "text":
+          if (content.type == "slider") {
+            content!.value.currentValue = content.value.sliderValueDefault;
+          }
+          contentArr.push(content);
+          break;
+        case "static_image":
+          console.log(content);
+          setMedia(content);
+          break;
```

Comment:
There is no case for animation in this version is this addition documented in another story or somewhere else?